sdk/cosmos/azure-cosmos/samples/change_feed_management.py

```diff
@@ -73,6 +73,52 @@ def read_change_feed_with_start_time(container, start_time):
     print('\nFinished reading all the change feed from start time of {}\n'.format(time))
 
 
+def read_change_feed_with_continuation(container, continuation):
+    print('\nReading change feed from continuation\n')
+
+    # You can read change feed from a specific continuation token.
+    # You must pass in a valid continuation token.
+    response = container.query_items_change_feed(continuation=continuation)
+    for doc in response:
+        print(doc)
+
+    print('\nFinished reading all the change feed from continuation\n')
+
+def delete_all_items(container):
+    print('\nDeleting all item\n')
+
+    for item in container.query_items(query='SELECT * FROM c', enable_cross_partition_query=True):
+        # Deleting the current item
+        print(item)
```

Comment:
no need to print the item when deleting/ creating if we're about to see it in the CF